src/optim_problems/run_opt.jl

```diff
@@ -59,8 +59,8 @@ what you can add to the opt_config:
 function run_opt(ts_data::ClustData,
                     opt_data::OptDataCEP,
                     opt_config::Dict{String,Any},
-                    fixed_design_variables::Dict{String,OptVariable};
-                    solver::Any=CbcSolver(),
+                    fixed_design_variables::Dict{String,OptVariable},
+                    optimizer::DataType;
```

Comment:
an optimizer is required now

Reply:
we would need to be dependent on package Cbc then and include `using Cbc` I don't think that's so elegant

Reply:
We would then be dependent on another package by `using Cbc` I think that's not so elegant

Reply:
Yep, agreed, that makes sense. That was a nice feature in the old JuMP that doesn't work anymore with optimizer I guess...